fairlearn/reductions/_exponentiated_gradient/_lagrangian.py

```diff
@@ -38,9 +38,12 @@ class _Lagrangian:
     opt_lambda : bool
         indicates whether to optimize lambda during the calculation of the
         Lagrangian; optional with default value True
+    sample_weight_key : str
+        Argument to supply `sample_weight` in `estimator.fit`.
```

Comment:
Can I suggest "Name of the argument to `estimator.fit()` which supplies the sample weights (defaults to `sample_weight`)"

Reply:
I think we should change `sample_weight_key` to `sample_weight_name`.